store/src/server/api.rs

```diff
@@ -388,10 +388,10 @@ fn invalid_argument<E: core::fmt::Debug>(err: E) -> Status {
 }
 
 #[instrument(target = "miden-store", skip_all, err)]
-fn validate_nullifiers(nullifiers: &[generated::digest::Digest]) -> Result<Vec<RpoDigest>, Status> {
+fn validate_nullifiers(nullifiers: &[generated::digest::Digest]) -> Result<Vec<Nullifier>, Status> {
     nullifiers
         .iter()
-        .map(TryInto::try_into)
+        .map(|digest_pb| Ok(Nullifier::from(RpoDigest::try_from(digest_pb)?)))
```

Comment:
Instead of going through `RpoDigest` - should we just implement `From<digest::Digest> for Nullifier` in the `miden-node-proto` crate?

Reply:
We actually have this implemented, my bad. Thanks for noticing!